frontend/src/component/project/Project/ProjectHealth/ReportTable/ReportTable.tsx

```diff
@@ -1,35 +1,35 @@
-import { useMemo } from 'react';
+import { useMemo } from "react";
 import {
     IEnvironments,
     IFeatureToggleListItem,
-} from 'interfaces/featureToggle';
-import { TablePlaceholder, VirtualizedTable } from 'component/common/Table';
-import { PageContent } from 'component/common/PageContent/PageContent';
-import { SearchHighlightProvider } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
-import { PageHeader } from 'component/common/PageHeader/PageHeader';
-import { sortTypes } from 'utils/sortTypes';
+} from "interfaces/featureToggle";
+import { TablePlaceholder, VirtualizedTable } from "component/common/Table";
+import { PageContent } from "component/common/PageContent/PageContent";
+import { SearchHighlightProvider } from "component/common/Table/SearchHighlightContext/SearchHighlightContext";
+import { PageHeader } from "component/common/PageHeader/PageHeader";
+import { sortTypes } from "utils/sortTypes";
 import {
     useFlexLayout,
     useGlobalFilter,
     useSortBy,
     useTable,
-} from 'react-table';
-import { useMediaQuery, useTheme } from '@mui/material';
-import { FeatureSeenCell } from 'component/common/Table/cells/FeatureSeenCell/FeatureSeenCell';
-import { FeatureTypeCell } from 'component/common/Table/cells/FeatureTypeCell/FeatureTypeCell';
-import { FeatureNameCell } from 'component/common/Table/cells/FeatureNameCell/FeatureNameCell';
-import { DateCell } from 'component/common/Table/cells/DateCell/DateCell';
-import { FeatureStaleCell } from 'component/feature/FeatureToggleList/FeatureStaleCell/FeatureStaleCell';
-import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
-import { Search } from 'component/common/Search/Search';
-import { ReportExpiredCell } from './ReportExpiredCell/ReportExpiredCell';
-import { ReportStatusCell } from './ReportStatusCell/ReportStatusCell';
-import { formatStatus, ReportingStatus } from './ReportStatusCell/formatStatus';
-import { formatExpiredAt } from './ReportExpiredCell/formatExpiredAt';
-import { useConditionallyHiddenColumns } from 'hooks/useConditionallyHiddenColumns';
-import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
-import { FeatureEnvironmentSeenCell } from 'component/common/Table/cells/FeatureSeenCell/FeatureEnvironmentSeenCell';
-import useFeatureTypes from 'hooks/api/getters/useFeatureTypes/useFeatureTypes';
+} from "react-table";
```

Comment:
I think this should be single quotes

Reply:
biome is double quotes

Reply:
No, biome is single quotes https://github.com/Unleash/unleash/blob/784d38c230fb345599867ece65ce800df545a1d4/biome.json#L84 but I'm curious why the linter check was ok: https://github.com/Unleash/unleash/actions/runs/6405856819/job/17389296240#step:7:6 @chriswk can I borrow your eyes for a second?